backend/api/models/ComplianceReportSchedules.py

```diff
@@ -151,3 +154,123 @@ class ScheduleBRecord(Model):
 
     class Meta:
         db_table = 'compliance_report_schedule_b_record'
+
+
+class ScheduleD(Model):
+    class Meta:
+        db_table = 'compliance_report_schedule_d'
+
+
+class ScheduleDSheet(Model):
+    schedule = models.ForeignKey(
+        ScheduleD,
+        related_name='sheets',
+        on_delete=models.PROTECT,
+        null=False
+    )
+
+    fuel_type = models.ForeignKey(
+        ApprovedFuel,
+        on_delete=models.PROTECT,
+        null=False
+    )
+
+    fuel_class = models.ForeignKey(
+        FuelClass,
+        on_delete=models.PROTECT,
+        null=False
+    )
+
+    feedstock = models.CharField(
+        max_length=100,
+        blank=True,
+        null=True
+    )
+
+    class Meta:
+        db_table = 'compliance_report_schedule_d_sheet'
+
+
+class ScheduleDSheetInput(Model):
+    sheet = models.ForeignKey(
+        ScheduleDSheet,
+        related_name='inputs',
+        on_delete=models.PROTECT,
+        null=False
+    )
+
+    worksheet_name = models.CharField(
+        max_length=100,
+        blank=True,
+        null=True
+    )
+    cell = models.CharField(
+        max_length=100,
+        blank=True,
+        null=True
+    )
+    value = models.CharField(
+        max_length=100,
+        blank=True,
+        null=True
+    )
+    units = models.CharField(
```

Comment:
Would there be any benefit to connecting to the unit_of_measure table for these values? It would help keep the data consistent. Might be low risk, but just wanted to mention it.